themes/base/templates/storyBlockTextEdit.handlebars

```diff
@@ -1,41 +1,25 @@
-<div lang="{{localizedMessages.language}}" class="sjrk-st-block">
+<fieldset lang="{{localizedMessages.language}}" class="sjrk-st-block">
+    <legend class="fl-hidden-accessible">{{localizedMessages.message_textBlock}}</legend>
     {{#getIds "storySelectionCheckbox"}}
         <div class="sjrk-st-label-container fl-hidden-accessible">
-            <label for="$id">{{../localizedMessages.message_blockSelectionCheckboxLabel}}</label>
```

Comment:
The translation of `message_blockSelectionCheckboxLabel` in `themes/base/messages/storyBlockMessages_es.json` and `themes/base/messages/storyBlockMessages_pt.json` should be replaced by labels for text, audio, image, video.

Reply:
Indeed. Since we disabled the localization functions, the message bundles for other languages have fallen out of date. This may take some time to complete

Reply:
Should I file a Jira and address this at a later time, or would you prefer that I make these updates now? I don't speak Portuguese, so that might just be a matter of adding the keys with empty strings (@gtirloni contributed the `pt` bundle).

Reply:
You said the localization has been disabled. It might make sense to have this work go into that JIRA.